packages/theme/src/unified/UnifiedThemeProvider.tsx

```diff
@@ -35,6 +39,12 @@ export interface UnifiedThemeProviderProps {
   noCssBaseline?: boolean;
 }
 
+// See https://mui.com/x/migration/migration-data-grid-v4/#using-mui-core-v4-with-v5
+const generateV4ClassName = createGenerateClassName({
+  disableGlobal: true,
+  seed: 'mui', // using a slightly shorter prefix than suggested in the docs
+});
+
```

Comment:
I'm wondering if `/unified/MuiClassNameSetup.ts` can be removed then. Probably not as we are still using `@mui/styles` which is also `jss`, right?

I might have taken the wrong conclusion from this [thread](https://github.com/backstage/backstage/pull/15484#discussion_r1066689862) for the `ClassNameGenerator`. It might make sense to investigate further if & how to remove `@mui/styles`/`jss` Styles Engine from our setup